src/View/Home/index.html.twig

```diff
@@ -1,10 +1,24 @@
 {% extends 'layout.html.twig' %}
-
 {% block classBody %}indexBackground
 {% endblock %}
+
 {% block content %}
-	<section class="indexSection">
-		<p class="logotxt">goldfish</p>
-		<img src="/assets/images/goldfish.png" class="logoimg" alt="logo goldfish">
-	</section>
+	<main class="container">
+		<div class="row">
+			<div class="d-flex flex-column justify-content-center">
+				<div class="indexSection">
+					<p class="logotxt">goldfish</p>
+					<img src="/assets/images/goldfish.png" class="logoimg" alt="logo goldfish">
+				</div>
+			</div>
+		</div>
+		<div class="row fixed-bottom mb-5">
+			<div class="form-signin">
+				<h3 class="baseline">Grâce à lui, vous ne passerez plus à côté des concerts de vos artistes favoris</h3>
+				<div class="button-connexion d-flex justify-content-center">
+					<button type="submit" class="connexionbutton btn btn-danger btn-md mb-5 d-flex justify-content-center">Connexion</button>
```

Comment:
"connexionbutton" 

no corresponding class in css.

Reply:
remove connexionbutton on class ok

Reply:
No not ok